grails-app/services/org/pih/warehouse/inventory/CycleCountProductAvailabilityService.groovy

```diff
@@ -202,18 +204,18 @@ class CycleCountProductAvailabilityService {
         // Output fields
         CycleCountItemsForRefresh cycleCountItemsForRefresh
 
-        CycleCountRefreshState(CycleCount cycleCount) {
+        CycleCountRefreshState(CycleCount cycleCount, Integer countIndex) {
             this.cycleCount = cycleCount
 
             // We only want to refresh the items of the most recent count. We do this because previous counts are
             // already completed, so their data should be considered static.
```

Comment:
can you update the comment to match your change? Something like:
```
// We only want to refresh the items of the most recent count. We do this because previous counts are
// already completed, so their data should be considered static. We allow countIndex to be specified
// for the case where the current count index has no items (QoH has been zeroed out for all items)
// and so we can't assume currentCountIndex == maxCountIndex.
```